edit-post/README.md

```diff
@@ -127,3 +128,93 @@ const MyPluginPostStatusInfo = () => (
 ```
 
 
+### `PluginPrePublishPanel`
+
+Renders provided content to the pre-publish side panel in the publish flow (side panel that opens when a user first pushes "Publish" from the main editor).
+
+_Example:_
+
+```jsx
+const { __ } = wp.i18n;
+const { PluginPrePublishPanel } = wp.editPost;
```

Comment:
Shouldn't these APIs just be `EditorPrePublishPanel` instead of Plugin...?

Reply:
We prefixed all other occurrences with `Plugin` in the past, this just keeps the convention.

Reply:
See the list of components in https://github.com/WordPress/gutenberg/blob/efa8098979a45127d7ac74ec0df600496c19bdac/edit-post/README.md

Reply:
Yes, but I'm not sure it makes a lot of sense for every piece. :) `PluginSidebar` and the plugin menu item make sense because their entire purpose is plugin based.

Reply:
I'm not attached to this, we can update.
What about `PluginPostStatustInfo`, should we also rename to `EditorPostStatusInfo`?

Reply:
I did some more research. We already have `PostPublishPanel`. See:
https://github.com/WordPress/gutenberg/blob/master/editor/components/post-publish-panel/index.js#L66
which uses `editor-post-publish-panel` class name. It might be a bit confusing. Maybe we should keep this `Plugin` prefix to make it clear that this is meant to be used by plugins.

Reply:
Yes, that's fine, was just a comment because it felt a bit weird to go with Plugin as a generic prefix across the board.